scripts/loadtest-scripts/prepare_loadtests.py

```diff
@@ -112,8 +109,10 @@ def main():
     exercise = get_collection_exercise()
     exercise_id = exercise['id']
 
-    if sdc.collection_exercises.get_state(exercise_id) in ['LIVE', 'READY_FOR_LIVE']:
-        logging.info('Quitting: The collection exercise has already been executed.')
+    if sdc.collection_exercises.get_state(exercise_id) in ['LIVE',
+                                                           'READY_FOR_LIVE']:
+        logging.info(
```

Comment:
Should this go to stderr?

Reply:
At this point it's un-decided on what the behaviour will be when this occurs. It's dependant on whether load tests will be runnable or not.